infra/recipes/k8s/environments/azure/sandbox/housetables/values.yaml

```diff
@@ -0,0 +1,34 @@
+namespace: "default"
+
+openhouseClusterTag: "openhouse"
+
+clusterPropertiesPath: "config/azure/sandbox/cluster.yaml"
+
+htsService:
+  replicaCount: 1
+  image:
+    repository: "containerregistryopenhouse.azurecr.io/openhouse-housetables-service"
```

Comment:
The name `containerregistryopenhouse` should be a variable.

Reply:
True good catch

Reply:
Where did you put this variable or it is not needed?

Reply:
Yes I ended up setting it in the helm_release: https://github.com/kmcclenn/openhouse/blob/services_deployed_helm/infra/recipes/terraform/azure/modules/helm_release/main.tf#L32,L36